pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -92,3 +115,88 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
+
+type ScaleREST struct {
+	registry statefulset.Registry
+}
+
+// ScaleREST implements Patcher
+var _ = rest.Patcher(&ScaleREST{})
+
+// New creates a new Scale object
+func (r *ScaleREST) New() runtime.Object {
+	return &extensions.Scale{}
```

Comment:
> In order to actually work, there has to be an object in the extensions category, so this storage basically doesn't matter.

This won't do.  We should not just create another "dummy" resource that sits in the extensions group.  That is an indication that the polymorphic nature of the endpoitn has failed.